catalyst_voices/packages/catalyst_voices_models/lib/src/key_derivation.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:typed_data';
+
+import 'package:catalyst_voices_models/src/seed_phrase.dart';
+import 'package:convert/convert.dart';
+import 'package:ed25519_hd_key/ed25519_hd_key.dart';
+
+class Ed25519KeyPair {
```

Comment:
A class called Ed25519KeyPair should already be defined in catalyst_cardano_serialization package but that's not where we should add this code.

Temporarily I would put it inside SeedPhrase as a non-static method, the return type should be an instance of Ed25519KeyPair from catalyst_cardano_serialization package.